paparazzi/paparazzi/src/main/java/app/cash/paparazzi/internal/PaparazziCallback.kt

```diff
@@ -99,16 +99,19 @@ internal class PaparazziCallback(
     return viewConstructor.newInstance(*constructorArgs)
   }
 
-  override fun resolveResourceId(id: Int): ResourceReference? = projectResources[id]
+  override fun resolveResourceId(id: Int): ResourceReference? =
+    projectResources[id] ?: dynamicResourceIdManager.findById(id)
 
   override fun getOrGenerateResourceId(resource: ResourceReference): Int {
     // Workaround: We load our resource map from fields in R.class, which are named using Java
     // class conventions.  Therefore, we need to similarly transform style naming conventions
     // that contain periods (e.g., Widget.AppCompat.TextView) to avoid false lookup misses.
     // Long-term: Perhaps parse and load resource names from file system directly?
-    val resourceKey =
-      if (resource.resourceType == STYLE) resource.transformStyleResource() else resource
-    return resources[resourceKey] ?: 0
+    val resourceKey = when (resource.resourceType) {
```

Comment:
nit: Since we are only checking one type I think we should leave it as the if statement

Reply:
reverted here: eb5cbccaded390d4f2ab20b37920b1fd36ca4614